test/trento/discovery/policies/cluster_policy_test.exs

```diff
@@ -9,6 +9,7 @@ defmodule Trento.Discovery.Policies.ClusterPolicyTest do
   require Trento.Enums.Provider, as: Provider
 
   require Trento.Clusters.Enums.HanaArchitectureType, as: HanaArchitectureType
+  require Trento.Clusters.Enums.HanaScenario, as: HanaScenario
```

Comment:
We need some new test for cost-optimized, right?

Reply:
Added test + payload